### ISSUE TITLE
Add one more tab on event page named 'Country Details' + use CIA info

### ISSUE THREAD

**Issue by LucieGueuning** [MSFREACH/msf-reach#295]
Nothing to be display under this one for now. Only to show that we have considered their requirements (part of the ethnography outcomes).

**Comment by matthewberryman** [comment]
@LucieGueuning I have added this tab, if you're happy with the tab name and placeholder text then update milestone to 2a.2 otherwise let me know what to change it to and I will update and then update milestone for the rest.

**Comment by LucieGueuning** [comment]
combined country facts and MSF data (cf. comment of Natasha on Presentation)

**Comment by matthewberryman** [comment]
@LucieGueuning reminder to supply me with data for this. For reference, list of countries in SE Asia can be found here https://github.com/MSFREACH/msf-reach/blob/master/public/resources/js/country-to-language-mapping.json

**Comment by LucieGueuning** [comment]
MSF Internal Information (past country policy paper) / CIA Factsheet info extracted

**Comment by matthewberryman** [comment]
Still waiting on supply of MSF data

**Comment by LucieGueuning** [comment]
- being able to upload as format of files for MSF internal document
- CIA Factshheet info for each country

**Comment by matthewberryman** [comment]
Making this issue about tab + CIA only. Integration with sharepoint / upload / other a new issue https://github.com/MSFREACH/msf-reach/issues/637

**Comment by matthewberryman** [comment]
@LucieGueuning please give this a try in dev

**Comment by LucieGueuning** [comment]
please follow comments from email sent earlier
